install/migrations/2.7.8.php

```diff
@@ -14,6 +14,14 @@ public static function getReleaseNotes()
 
         public static function upgrade()
         {
+            // Add the log filter value
+            // This was added in 2.7.1, but the update process never took care of it
+            // Re: #2762
+            $filter = Symphony::Configuration()->get('filter', 'log');
+            if ($filter === null) {
+                Symphony::Configuration()->set('filter', E_ALL ^ E_DEPRECATED, 'log');
```

Comment:
I have a feeling this will get stringified or turned into an integer on saving. That's ok, I believe it'll still have the same effect, it just won't be saved _exactly_ as it's seen here.